Include path not found

I don't know why, but to build 1.8.0, I had to edit a few files and hard code the path to `ui_*.h` files, because I was getting failures like this:

```
[ 32%] Building CXX object CMakeFiles/birdtray_lib.dir/src/updatedownloaddialog.cpp.o
/home/user/src/birdtray/src/mailaccountdialog.cpp:8:34: fatal error: ui_mailaccountdialog.h: No such file or directory
compilation terminated.
```

Once I updated the `#include` lines to point to `/home/user/src/birdtray/build/ui_mailaccountdialog.h`, in this example, it built fine. I suspect this is an issue with my environment, but I was merely following the README.md instructions:

```
mkdir build
cd build
cmake -DCMAKE_BUILD_TYPE=Release ..
cmake --build .
```

I don't recall having to do this for 1.7.0.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,6 +54,10 @@ endif(WIN32)
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTOUIC ON)
 add_definitions(-DQT_DEPRECATED_WARNINGS)
+if(${CMAKE_VERSION} VERSION_LESS "3.8.0")
+    # AUTO_UIC places generated header files in CMAKE_CURRENT_BINARY_DIR before 3.8.0.
+    set(CMAKE_INCLUDE_CURRENT_DIR ON)
+endif()
 
 set(SOURCES
         src/colorbutton.cpp

